pyqtgraph/widgets/RawImageWidget.py

```diff
@@ -3,7 +3,9 @@
     from ..Qt import QtOpenGL
     from OpenGL.GL import *
     HAVE_OPENGL = True
-except ImportError:
+except Exception:
```

Comment:
I cast a broad net here because I don't really care what the error is--if either of these packages cannot be imported, then the behavior should always be to handle it gracefully. Don't want to find that the next pyopengl update raises some other unexpected exception.